apps/blog/_posts/2026/01/coding-agent/claude-md-agents-md.md

```
@@ -0,0 +1,65 @@
+- **Update it constantly** - if you keep correcting Claude on the same thing, that's a signal it should be in CLAUDE.md. Just say `remember this to CLAUDE.md`
+- **Subdirectory CLAUDE.md files** - this is useful for monorepos, lazy loaded when Claude is actively working in that part of the codebase (e.g. `apps/home/CLAUDE.md`, `apps/blog/CLAUDE.md`, etc).
+
+[AGENTS.md](https://agents.md) serves a similar purpose. If you use both Claude Code and other coding agents (like Codex, Cursor), create a symlink so they share the same instructions:
+
+```bash
+ln -s CLAUDE.md AGENTS.md
+```
+
+or put instructions in `AGENTS.md` (an open standard) and reference it from `CLAUDE.md`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)